packages/components/src/slot-fill/bubbles-virtually/slot.js

```diff
@@ -41,7 +49,11 @@ function Slot(
 	} );
 
 	return (
-		<Component ref={ useMergeRefs( [ forwardedRef, ref ] ) } { ...props } />
+		<View
```

Comment:
Do we need to use the `View` component here? Seems to me that the original `div` was fine, and that now we're just adding more indirection and complexity. Is there any tangible benefit from the fact that the `div` is now styled with Emotion?

Reply:
@jsnajdr Thanks. 

In the original Pull Request, I changed to View to avoid `error: TS2590`, which I am trying to address in this PR.

https://github.com/WordPress/gutenberg/pull/51350#discussion_r1248284719

Reply:
I see, so it's not because of `View` functionality, but to solve a typing problem. Hopefully in future, when the rename from JS to TS is done, we can find a way to go back to simple `div`.